pdns/remote_logger.hh

```diff
@@ -118,3 +125,6 @@ private:
   LockGuarded<RuntimeData> d_runtime;
   std::thread d_thread;
 };
+
+// Helper to be defined by main program: queue data and log based on return value of queueData()
+void remoteLoggerQueueData(RemoteLoggerInterface&, const std::string&);
```

Comment:
I'm not sure I like this helper being defined here, I would prefer to have it static in dnsdist-lua.cc, for example. It does not matter much, but it might make unit testing that class harder, etc.